kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -82,6 +83,32 @@ def create_filters(self):
             self.J_pad, self.J, self.Q, self.T,
             r_psi=self.r_psi, sigma0=self.sigma0, alpha=self.alpha)
 
+    def scattering(self, x):
```

Comment:
trés cool

Reply:
wait till this is used to implement `.meta()` and `.out_size()` automatically (thanks to `DryBackend()`) ☺️